src/service/standard-actions-impl.js

```diff
@@ -88,14 +103,62 @@ export class StandardActions {
    */
   handleHide(invocation) {
     const target = dev().assertElement(invocation.target);
+
     this.resources_.mutateElement(target, () => {
       if (target.classList.contains('i-amphtml-element')) {
         target./*OK*/collapse();
       } else {
         toggle(target, false);
+        target.setAttribute('hidden', '');
```

Comment:
We still support IE10 though, right? And I think we cut off at Chrome 37. Perhaps we can add it just in case to `amp.css`? I.e. 
```
[hidden] {
  display: none !important;
}
```
?